autoload/open_url.vim

```diff
@@ -5,7 +5,7 @@ function! open_url#open(url)
   elseif has('mac') || has('macunix') || has('gui_macvim')
     exe 'silent !open "'.url.'"'
   else
-    exe 'silent !'.g:open_url_browser_default url
+    exe 'silent! !'.g:open_url_browser_default.' "'.url.'" &> /dev/null &'
```

Comment:
I am not sure if `$> /dev/null` is required, neither is the trailing `&` to send this to background.

Reply:
Those two are necessary, when plugin is opening the first instance of browser (I use `firefox`, I don't know how it is with other browsers, but compatibility with one of the most popular isn't a bad thing, right? ;) )

Without `&` I won't be able to go back to Vim and continue editing while having browser open.

Without `&> /dev/null` buffers get polluted when closing browser with such messages:
```
###!!! [Parent][DispatchAsyncMessage] Error: PLayerTransaction::Msg_ReleaseLayer Processing error: message was deserialized, but the handler returned false (indicating failure)          
                                                                                                                                                                                                                                      
                                                                                                                                                                                                                            [Parent 21844, Gecko_IOThread] WARNING: pipe error (42): Connection reset by peer: file /build/firefox/src/firefox-68.0.1/ipc/chromium/src/chrome/common/ipc_channel_posix.cc, line 358                                                                  
                                                                                                                                                                            [Parent 21844, Gecko_IOThread] WARNING: pipe error (98): Connection reset by peer: file /build/firefox/src/firefox-68.0.1/ipc/chromium/src/chrome/common/ipc_channel_posix.cc, line 358                                                                                                                          
                                                                                                                                                                                                                                             
                                                                                                                     ###!!! [Parent][RunMessage] Error: Channel closing: too late to send/recv, messages will be lost                          
```
and there is a need for another `redraw!` but I think this approach is easier than detecting if browser was closed.